release/gcb/gcb_lib.sh

```diff
@@ -95,8 +95,24 @@ function make_istio() {
   sha256sum "${ISTIO_OUT}/istio-sidecar.deb" > "${OUTPUT_PATH}/deb/istio-sidecar.deb.sha256"
   cp        "${ISTIO_OUT}/istio-sidecar.deb"   "${OUTPUT_PATH}/deb/"
   cp        "${ISTIO_OUT}"/archive/istio-*z*   "${OUTPUT_PATH}/"
-  cp        "${ISTIO_OUT}"/archive/istioctl*.tar.gz "${OUTPUT_PATH}/"
-  cp        "${ISTIO_OUT}"/archive/istioctl*.zip "${OUTPUT_PATH}/"
+  
+  ISTIOCTL_TARS=$(ls "${ISTIO_OUT}"/archive/istioctl*.tar.gz)
```

Comment:
ISTIOCTL_TARS+=$(ls "${ISTIO_OUT}"/archive/istioctl*.zip)
then you can remove the next for loop

Reply:
Addressed in the latest update.